### PR TITLE
CSS clean up + show error when no prediction came through

### PR DESCRIPTION
**Pull request by justinebreuch**
## Description

Addresses: Add error handling on any API failures

This is just covering all failures right now in the prediction panel. See screenshot. 

Also a bunch of css cleanup to conform to copper.

## Testing and Reviewing
Right now my API is always failing so this was a good time to do the PR:

Screen Shot 2022-08-19 at 11 22 59 AM
 checklists for how someone can review this PR to verify it's working as intended_

### CODE REVIEW

File: client/src/views/ScorecardView.vue [hunk 1]
```diff
@@ -72,9 +69,12 @@ export default defineComponent({
...
 </script>
 
-<style scoped>
+<style scoped lang='scss'>
+@import '../assets/styles/global.scss';
+@import '@blueconduit/copper/scss/01_settings/design-tokens';
+
 .actions-to-take {
-  background-color: #E1F5FE;
+  box-shadow: 0px 0px 8px #757575;
```
Comment (beekley) [inline client/src/views/ScorecardView.vue:77]: Would one of [the shadows from Copper](https://github.com/BlueConduit/copper/blob/main/packages/copper/scss/01_settings/_design-tokens.scss#L66) work here?

**Review by beekley — APPROVED** [review]

File: client/src/assets/styles/global.scss [hunk 2]
```diff
@@ -1,3 +1,8 @@
+@use '@blueconduit/copper/scss/01_settings/design-tokens' as dt;
```
Comment (kailamjeter) [inline client/src/assets/styles/global.scss:1]: nit can you do as * instead so we don't have to prefix with dt.

File: client/src/assets/styles/global.scss [hunk 3]
```diff
@@ -71,6 +72,21 @@ $navy-blue: #2553A0;
...
+  display: flex;
+}
+
+@mixin container-column {
+  display: flex;
+  flex-direction: column;
+}
+
+@mixin centered-container {
+  @include container-column;
```
Comment (kailamjeter) [inline client/src/assets/styles/global.scss:86]: can we keep this agnostic of direction?

**Review by kailamjeter — APPROVED** [review]

File: client/src/assets/styles/global.scss (hunk 2 above)
Comment (justinebreuch) [reply client/src/assets/styles/global.scss:1]: Done!

File: client/src/assets/styles/global.scss (hunk 3 above)
Comment (justinebreuch) [reply client/src/assets/styles/global.scss:87]: That's a good idea

File: client/src/views/ScorecardView.vue (hunk 1 above)
Comment (justinebreuch) [reply client/src/views/ScorecardView.vue]: Obsolete!

**Pull request merged**